docs/src/components/ExampleCode.js

```diff
@@ -61,28 +68,29 @@ export default function ExampleCode({ code, name }: {| code: string, name: strin
             borderStyle="sm"
             display="flex"
             overflow="hidden"
+            maxHeight={!expanded ? CODE_EXAMPLE_HEIGHT : undefined}
             position="relative"
             ref={codeExampleRef}
             rounding={2}
-            {...(expanded ? {} : { maxHeight: CODE_EXAMPLE_HEIGHT, overflow: 'hidden' })}
           >
             <Box
               flex="grow"
               onFocus={() => {
                 setExpanded(true);
               }}
             >
-              {/* We can not pass in an id for LiveEditor which links to the underlying text area */}
-              {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-              <label>
-                <LiveEditor
-                  className="live-editor-pane"
```

Comment:
This isn't an available prop, and we don't set this class

Reply:
This was being used in commands.js to avoid Cypress testing the accessibility of the code editor (there was an issue with it detecting the right background color IIRC), so I'd just keep an eye out if those tests still pass. This class name was showing up on the outer div

Reply:
Ah yup, looks like they're all failing again